packages/vitest/src/utils/source-map.ts

```diff
@@ -83,6 +83,11 @@ export function parseStacktrace(e: ErrorWithDiff, full = false): ParsedStack[] {
         file,
         line: parseInt(lineNumber),
         column: parseInt(columnNumber),
+        sourcePos: {
```

Comment:
After a little but of retrospective, I think the change in `api/setup` can stay, but this change is useless in the grand scheme of things, and VS Code extension should rewrite their code.

Reply:
Fair that make sense 👍, I can a PR there to make it default to use the actual values if `sourcePos` is undefined 👍. We probably should remove `sourcePos` from the type though, from what I've seen across the repo it doesn't seem to be set anywhere anymore.

Do you want me to open a PR that removes it accross the codebase?

Reply:
Just updated the PR to not include that change 👍!

Reply:
> Fair that make sense 👍, I can a PR there to make it default to use the actual values if `sourcePos` is undefined 👍. We probably should remove `sourcePos` from the type though, from what I've seen across the repo it doesn't seem to be set anywhere anymore.
> 
> Do you want me to open a PR that removes it accross the codebase?

You are right, we don't use it anymore. If it's ok with you, we can remove `sourcePos` in this pr.

Reply:
Done ✅!

Reply:
Should be good now! A test isn't passing but I think it's unrelated to this PR (some web worker test failing to start on Windows)